### PR TITLE
#8644 Default service for background addition, documentation update, actions whitelist extension

### PR DESCRIPTION
**Pull request by alex-fko**
## Description
This PR aims to resolve:
https://github.com/geosolutions-it/MapStore2/issues/8644#issuecomment-1266640834

- Background will use default service if no service is specified in parameter value;
- Documentation is updated with information about scheduled search action (scheduled map info);
- `SEARCH:SCHEDULE_SEARCH_WITH_FILTER` is added to the initial actions whitelist.

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [x] Other... Please describe: Documentation update

## Issue

**What is the current behavior?**

https://github.com/geosolutions-it/MapStore2/issues/8644#issuecomment-1266640834

**What is the new behavior?**
As per description

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: docs/developer-guide/map-query-parameters.md [hunk 1]
```diff
@@ -224,17 +224,18 @@ Allows to dynamically add background to the map and activate it.
...
 
+`#/viewer/openlayers/new?background=layer2`
+
 Where:
 
-- `Sentinel` & `layer1` are layer names,
-- `service` is the service name providing layer data. Service name is optional,
-`default_map_backgrounds` will be used if it is omitted.
+- `Sentinel`, `layer1`, `layer2` are layer names,
+- `service`, `default_map_backgrounds` are the service names providing layer data. Service name is optional. If no service is provided, the default service will be used.
```
Comment (offtherailz) [inline docs/developer-guide/map-query-parameters.md:236]:
```suggestion
- `service`, `default_map_backgrounds` are the service names providing layer data. Service name is optional. If no service is provided, the default service of the catalog will be used.
```

File: docs/developer-guide/map-query-parameters.md [hunk 2]
```diff
@@ -214,7 +214,7 @@ GET: `#/viewer/openlayers/new?addLayers=layer1;service&mapinfo=layer1&mapInfoFil
 Where:
 
 - `layer1` is layer name. 
-- `service` is the service name providing layer data. Service name is optional. If missing the default.
+- `service` is the service name providing layer data. Service name is optional. If no service is provided, the default service will be used.
```
Comment (offtherailz) [inline docs/developer-guide/map-query-parameters.md:217]:
```suggestion
- `service` is the service name providing layer data. Service name is optional. If no service is provided, the default service of the catalog will be used.
```

File: docs/developer-guide/map-query-parameters.md [hunk 3]
```diff
@@ -309,6 +310,23 @@ The MapStore invocation URL above executes the following operations:
 
 For more details check out the [searchLayerWithFilter](https://mapstore.geosolutionsgroup.com/mapstore/docs/#actions.search.exports.searchLayerWithFilter) in the framework documentation
 
+#### Scheduled Map Info
+
+It works similarly to the `Map Info` action, but supports delaying of the search execution up to the moment when layer is added to the map.
+This behavior is used when search should be applied to the dynamically added layer:
```
Comment (offtherailz) [inline docs/developer-guide/map-query-parameters.md:316]:
```suggestion
This behavior is used when search should be applied to the dynamically added layer (e.g. using `addLayer` parameter) :
```

**Review by offtherailz — APPROVED** [review]

**Comment by offtherailz** [comment]
I applied some documentation improvemetns. @alex-fko please check

**Comment by alex-fko** [comment]
@offtherailz looks good, thanks

**Comment by tdipisa** [comment]
@ElenaGallo please test this on DEV and let us know if we can proceed with the backport of all things to 2022.02.xx

**Pull request merged**